src/yz_solrq_drain_mgr.erl

```diff
@@ -65,35 +65,46 @@ init([]) ->
     schedule_tick(),
```

Comment:
Not related to this changeset, but just something I noticed during reviews: the implementation of `schedule_tick` down below uses the `?MODULE` macro to send a message to ourselves. This will work fine, but we already define a `?SERVER` macro to avoid the code being coupled to the module name, so we should probably use that instead.